tpm2/test/symmetric_encryption_test.go

```diff
@@ -0,0 +1,291 @@
+package tpm2test
+
+import (
+	"bytes"
+	"crypto/aes"
+	"crypto/rand"
+	"io"
+	"testing"
+
+	. "github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpm2/transport"
+	"github.com/google/go-tpm/tpm2/transport/simulator"
+)
+
+const maxDigestBuffer = 1024
+
+func TestAESEncryption(t *testing.T) {
+	theTpm, err := simulator.OpenSimulator()
+	if err != nil {
+		t.Fatalf("could not connect to TPM simulator: %v", err)
+	}
+	t.Cleanup(func() {
+		if err := theTpm.Close(); err != nil {
+			t.Errorf("%v", err)
+		}
+	})
+
+	createPrimaryCmd := CreatePrimary{
```

Comment:
Could you simplify this test by making the AES key a primary key?

Reply:
done;  (just a habit to make make primary and then key)